tests/api/entities/get/by_isbns.test.ts

```diff
@@ -15,6 +19,64 @@ describe('entities:get:by-isbns', () => {
     should(res.notFound).not.be.ok()
   })
 
+  describe('wikidata editions', () => {
+    it('should find Wikidata edition entities by its isbn uri', async () => {
```

Comment:
The test is (the only) red.

```
     TypeError: Cannot read properties of undefined (reading 'should')
      at Context.<anonymous> (tests/api/entities/get/by_isbns.test.ts:28:32)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

Is something wrong with my config ? Is it green on your side ?
The data i have is quite new (ie. cache was busted)

Reply:
I pushed a debug commit: can you fetch, rerun this test and report the server and tests log lines starting with "🚀"?

Reply:
> logs above shows there is an inv local entity with this ISBN, which I guess you don't have in your local db

I assumed you tried with an empty `entities-tests` database: yes, if there is already a local entity with that ISBN, that test will fail due to the priority now being given to the local entity (since 9fb9a80cb). Can you retry with an empty  `entities-tests`  db?

Reply:
'invp:P1' is a property based on same as entities only. Well, those comments helped to understand the codebase. Thanks!